.eslintrc.json

```diff
@@ -6,7 +6,7 @@
   "extends": [
     "eslint:recommended",
     "plugin:react/recommended",
-    "plugin:react-hooks/recommended",
```

Comment:
Was removing `plugin:react-hooks/recommended` on purpose? Might it cause problems, if we use hooks?

Reply:
I think you should return it.

Reply:
Thank you, I returned it.